bin/develop.js

```diff
@@ -0,0 +1,22 @@
+import program from 'commander'
```

Comment:
Reminder that this code needs to be ES5/commonjs as Babel doesn't work when code is globally installed https://phabricator.babeljs.io/T1889 https://github.com/gatsbyjs/gatsby/issues/54